### DISCUSSION TITLE
Speech Recognition doesn't work after losing focus and refocus into app.

### DISCUSSION THREAD

**Discussion by john-jeffri** [microsoft/MixedRealityToolkit-Unity#10519] [Q&A] [answered]
Hi, I'm using voice commands in my Unity app. And it works super well at the launch but when I lose focus and come back those voice commands doesn't work. Using Unity 2020.3.15f2 and MRTK 2.7.2.0. Any help will be really appreciated.

**Comment by john-jeffri** [comment, marked as answer]
`private void OnApplicationFocus(bool isFocused)
    {
        Debug.Log($"Is App focused? : {isFocused}");

        if (!isFocused)
        {
            PhraseRecognitionSystem.Shutdown();
        }
        else
        {
            PhraseRecognitionSystem.Restart();
        }
    }`

This fixed the issue.

**Reply by williamedu** [reply]
i have no words to describe how thankfull i am for your comment you truly saved my life with this, thank you so much man. god bless you.
